pb_plugins/dcsdkgen/type_info.py

```diff
@@ -0,0 +1,50 @@
+import json
+from os import environ
+
+
+class TypeInfo:
+    _default_types = {
+        1: "double",
+        2: "float",
+        4: "uint64_t",
+        5: "int",
+        8: "bool",
+        9: "std::string",
+        13: "uint32_t"
+    }
+
+    try:
```

Comment:
This is probably ugly, but I did not know how to do it otherwise. Open to propositions :-).

Reply:
Can't think of anything that's more elegant!